hack/test.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+DOWN=${DOWN:true}
+
+super_echo() {
+  echo ""
+  echo "**"
+  echo "**$1"
+  echo "**"
+}
+
+super_echo "WARNING: this script is experimental, run at your own risk!"
```

Comment:
I like how this prints but doesn't give you the opportunity to stop it.  Also, what is the risk?  Seems like a rather innocuous script.

Reply:
IDK maybe I'll remove this warning. Or at least reword it to be clear that it's writing stuff into /var/kubernetes (since that's what aws-iam-authenticator init and hack/local-up-cluster.sh does)